experimental/veggieseasons/lib/styles.dart

```diff
@@ -216,12 +216,20 @@ abstract class Styles {
 
   static const Color iconGold = Color(0xffdba800);
 
+  static const Color iconRed = Color(0xffff0000);
```

Comment:
Using a pure red for Apple is probably wrong. According to Apple's [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/color/) `systemRed` is R 255, G 59, B 48 on a light background, and R 255, G 69 and B 58 on a dark background.

Reply:
yes, I just changed it!